backend/src/daemon/daemon_thread.rs

```diff
@@ -166,6 +171,14 @@ impl ThreadClient {
         self.send(set_enum).await.and(Ok(()))
     }
 
+    pub async fn disable_refresh(&self) -> Result<(), String> {
+        self.send_noresp(SetEnum::DisableRefresh).await
+    }
+
+    pub async fn enable_refresh(&self) -> Result<(), String> {
+        self.send_noresp(SetEnum::EnableRefresh).await
+    }
+
```

Comment:
Refreshing is only done when the frontend invokes `.refresh()`, in a timer set in `src/main_window.rs`. So this should be done by disabling the call there, rather than modifying the backend.

Reply:
Ya I'm aware of that, I don't want to use the static variable everywhere, this seems fine to me

Reply:
Adding a command to a library to disable another command in that library, to work around the behavior in the application using that library doesn't seem great.

A simple `static` `AtomicBool` in the GUI crate would be fine, and involve less boilerplate. (And fewer additional conflicts with https://github.com/pop-os/keyboard-configurator/pull/169). Not the best way, but it can be done differently in a future Iced based UI.

Reply:
simple enough [72d6afc](https://github.com/pop-os/keyboard-configurator/pull/174/commits/72d6afc5d32c836279c09966ebc06c82cfa0eb8f)